### DISCUSSION TITLE
How can I add click_action

### DISCUSSION THREAD

**Discussion by aalinia** [kreait/firebase-php#638] [Q&A] [answered]
Is there any way to add `click_action` to the message for androids

**Comment by jeromegamez** [comment, marked as answer]
You can define platform specific configuration https://firebase-php.readthedocs.io/en/5.x/cloud-messaging.html#adding-target-platform-specific-configuration

Here's the full reference of which options you can set: https://firebase.google.com/docs/reference/fcm/rest/v1/projects.messages#androidnotification

**Reply by aalinia** [reply]
:pray: Thanks
